gulpfile.js

```diff
@@ -41,9 +45,12 @@ var paths = {
     'js/control/*.js',
     'js/index.js'
   ]),
-  styles: mainBowerFiles('**/*.css').concat('css/*.css'),
-  images: mainBowerFiles('**/*.+(png|gif|svg)'),
-  fonts: mainBowerFiles('**/font-awesome/fonts/*'),
+  styles: mainNpmFiles().filter(f => 
+    RegExp('.*\.css', 'i').test(f) &&
```

Comment:
```suggestion
    RegExp('.*\\.css', 'i').test(f) &&
```
As mentioned in my comment, I think the backslash needs also to be escaped.